<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpForm</class>
 <widget class="QWidget" name="SignUpForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>832</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="signUp_Male">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Male</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signUp_UploadPicture">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Upload Picture</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="signUp_Female">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Female</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>UserName:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signUp_PassWord">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="signUp_LastName">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="signUp_DateBirth">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>MM/dd/yyyy</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signUp_UserName">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="signUp_Submit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>PassWord:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FirstName:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>BirthDate:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LastName: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="signUp_FirstName">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="testresult">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>104</width>
     <height>291</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
